src/app.ts

```diff
@@ -1,24 +1,73 @@
-const Koa = require('koa');
-const cors = require('@koa/cors');
+#!/usr/bin/env node
 
-const { admin } = require('./firebaseAdmin');
-const { verifyFirebaseIdToken } = require('./verifyFirebaseIdToken');
+import path from 'path';
+
+import Koa from 'koa';
+import cors from '@koa/cors';
+import Router from 'koa-router';
+
+import { getAdmin } from './firebaseAdmin';
+import  { verifyFirebaseIdToken } from './verifyFirebaseIdToken';
+
+let admin;
+if (process.env.NODE_ENV === 'test' && process.argv.length > 2) {
```

Comment:
So we never need to create credentials in prod because we also run it in GC? Only for running raha-test locally? That's nice :)

Reply:
Yerp!